docs/rest_api.md

```diff
@@ -124,6 +124,19 @@ Possible Errors:
     * The network is connected to a node or headnode.
     * There are pending actions involving the network.
 
+### list_networks
+
+`GET /networks`
+
+List all networks
+
+The result contains a JSON dictionary of dictionaries, indexd by the name of the network. Then interior dictionaries are indexed by project and channel.
```

Comment:
typo: indexd = indexed.

Also - could you describe this using the terms "key" and "value", like we do in #537? Something like:

> The result contains a JSON dictionary of dictionaries. The first-level dictionary has keys equal to the names of the networks, with values being another dictionary containing the keys \`channel\` and \`project\`.